Modify CreateTrie Function and some menu visualized

AndyNg195 opened the pull request:
cập nhật tìm phổ biến dài nhất
chưa test

AndyNg195 committed:
updatecode(add):  find longest common

WhoamiSexy commented:
void longestcommonTrieString(Node* dad)
{
	string a = "";
	int thanhphan = 1, b;
	vector<string> temp;
	longestcommonTrieString(dad, thanhphan, a, temp);
	for (string ptr : temp)
	{
		b = stoi(ptr.substr(ptr.length() - 1));
		if (b > thanhphan)
		{
			thanhphan = b;
		}
	}

	cout << "Chuoi dai nhat: " << endl;
	for (string ptr : temp)
	{
		b = stoi(ptr.substr(ptr.length() - 1));
		if (b == thanhphan)
		{
			cout << ptr.substr(0, ptr.length() - 1) << endl;
		}
	}
}

AndyNg195 committed:
delete_file(doannhom12.cpp): change to main.cpp
add_file(main.cpp): same as doannhom12.cpp with addition of find LongestCommonSubstring function
add_file(word.txt): resources

AndyNg195 committed:
delete_file(doannhom12.exe): clear the remain
modified(main.cpp): move contents of main() to menu()

AndyNg195 committed:
modified(main.cpp):
    update: menu //console
    fix_function: createTrie()
